src/Role/ValueObjects/Permission.php

```diff
@@ -4,32 +4,81 @@
 
 namespace CultuurNet\UDB3\Role\ValueObjects;
 
-use ValueObjects\Enum\Enum;
-
-/**
- * Class Permission
- * @package CultuurNet\UDB3\Role\ValueObjects
- * @method static Permission AANBOD_BEWERKEN()
- * @method static Permission AANBOD_MODEREREN()
- * @method static Permission AANBOD_VERWIJDEREN()
- * @method static Permission ORGANISATIES_BEHEREN()
- * @method static Permission ORGANISATIES_BEWERKEN()
- * @method static Permission GEBRUIKERS_BEHEREN()
- * @method static Permission LABELS_BEHEREN()
- * @method static Permission VOORZIENINGEN_BEWERKEN()
- * @method static Permission PRODUCTIES_AANMAKEN()
- * @method static Permission FILMS_AANMAKEN()
- */
+use CultuurNet\UDB3\Model\ValueObject\String\Enum;
+
 class Permission extends Enum
 {
-    public const AANBOD_BEWERKEN = 'Aanbod bewerken';
-    public const AANBOD_MODEREREN = 'Aanbod modereren';
-    public const AANBOD_VERWIJDEREN = 'Aanbod verwijderen';
-    public const ORGANISATIES_BEHEREN = 'Organisaties beheren';
-    public const ORGANISATIES_BEWERKEN = 'Organisaties bewerken';
-    public const GEBRUIKERS_BEHEREN = 'Gebruikers beheren';
-    public const LABELS_BEHEREN = 'Labels beheren';
-    public const VOORZIENINGEN_BEWERKEN = 'Voorzieningen bewerken';
-    public const PRODUCTIES_AANMAKEN = 'Producties aanmaken';
-    public const FILMS_AANMAKEN = 'Films aanmaken';
+    public static function getAllowedValues(): array
+    {
+        return [
+            'Aanbod bewerken',
+            'Aanbod modereren',
+            'Aanbod verwijderen',
+            'Organisaties beheren',
+            'Organisaties bewerken',
+            'Gebruikers beheren',
+            'Labels beheren',
+            'Voorzieningen bewerken',
+            'Producties aanmaken',
+            'Films aanmaken',
+        ];
+    }
+
+    public static function getAllowedPermissions(): array
```

Comment:
Small suggestion, maybe a name like `getAll()` or `getAllPermissions()` is better for this method. Because the "Allowed" of "getAllowedValues()" is generic and exists on all enums and is in regards to what values are allowed on this specific enum type (fixed list of possible values). While "getAllowedPermissions()" can get confusing  because it might seem like those are the permissions allowed for a specific user or role.